src/memefactory/server/graphql_resolvers.cljs

```diff
@@ -1,16 +1,687 @@
+                           :join [:reg-entries [:= :reg-entries.reg-entry/address :param-changes.reg-entry/address]]
+                           :where [:= address :param-changes.reg-entry/address]})]
+    (log/debug "param-change query" sql-query)
+    sql-query))
+
+(defn search-param-changes-query-resolver [_ {:keys [:first :after] :as args}]
+  (log/debug "search-param-changes args" args)
+  (paged-query {:select [:*]
+                :from [:param-changes]
+                :join [:reg-entries [:= :reg-entries.reg-entry/address :param-changes.reg-entry/address]]}
+               first
+               (when after
+                 (js/parseInt after))))
+
+(defn user-query-resolver [_ {:keys [:user/address] :as args} context debug]
+  (log/debug "user args" args)
+  (let [sql-query (db/get {:select [:*]
+                           :from [:users]
+                           :where [:= address :users.user/address]})]
+    (log/debug "user query" sql-query)
+    (when-not (empty? sql-query)
+      sql-query)))
+
+(defn search-users-query-resolver [_ {:keys [:order-by :order-dir :first :after] :as args}]
+  (log/debug "search-users-query-resolver args" args)
+  (let [now (last-block-timestamp)
+        order-by-clause (when order-by (get {:users.order-by/address :user/address
+                                             :users.order-by/voter-total-earned :user/voter-total-earned
+                                             :users.order-by/challenger-total-earned :user/challenger-total-earned
+                                             :users.order-by/curator-total-earned :user/curator-total-earned
+                                             :users.order-by/total-participated-votes-success :user/total-participated-votes-success
+                                             :users.order-by/total-participated-votes :user/total-participated-votes
+                                             :users.order-by/total-created-challenges-success :user/total-created-challenges-success
+                                             :users.order-by/total-created-challenges :user/total-created-challenges
+                                             :users.order-by/total-collected-memes :user/total-collected-memes
+                                             :users.order-by/total-collected-token-ids :user/total-collected-token-ids
+                                             :users.order-by/total-created-memes-whitelisted :user/total-created-memes-whitelisted
+                                             :users.order-by/total-created-memes :user/total-created-memes}
+                                            (graphql-utils/gql-name->kw order-by)))
+        query (paged-query {:select [:*
+                                     [(sql/raw "(users.user_SLASH_voter_total_earned + users.user_SLASH_challenger_total_earned)") :user/curator-total-earned]
+                                     [{:select [:%count.*]
```

Comment:
These queries are all executed all the time, no matter what fields were actually requested, right? 
We'd probably need a way to run queries only for fields that are requested here, since it's sort of performance sensitive.